### PR TITLE
216 fix issue where pyjnius is loaded from global context

### PR DESCRIPTION
**Pull request by lfse-slafleur**

### CODE REVIEW

File: src/omotes_simulator_core/entities/assets/pyjnius_loader.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        jnius_config.add_classpath(os.path.join(path, "bin/jfxrt.jar"))
+        jnius_config.add_classpath(os.path.join(path, "bin/rosim-batch-0.4.2.jar"))
+
+        self.loaded_classes = {}
+
+    def load_class(self, classpath: str) -> JavaClass:
+        """Load a Java class.
+
+        If it has been loaded previously, the reference to the class will be laoded from cache.
```
Comment (MichielTukker) [inline src/omotes_simulator_core/entities/assets/pyjnius_loader.py:55]: typo

File: ci/linux/lint.sh
Comment (MichielTukker) [inline ci/linux/lint.sh:1]: empty file?

**Review by MichielTukker — APPROVED** [review]

File: ci/linux/lint.sh
Comment (lfse-slafleur) [reply ci/linux/lint.sh:1]: Only added +x permission on linux. No changes in the file.

**Comment by MichielTukker** [comment]
Added Ryvo for review (mostly FYI) as he's the original author of the Ates class

File: src/omotes_simulator_core/entities/assets/pyjnius_loader.py (hunk 1 above)
Comment (lfse-slafleur) [reply src/omotes_simulator_core/entities/assets/pyjnius_loader.py:55]: Apologies, I saw this one to late. Fixed here: https://github.com/Project-OMOTES/simulator-core/pull/218

**Comment by MichielTukker** [comment]
Also closes #158

**Pull request merged**